src/test/javascript/package.json

```diff
@@ -13,14 +12,14 @@
     "console.table": "^0.10.0",
     "eventsource": "^0.1.6",
     "ip": "^0.3.0",
-    "jasmine": "^2.6.0",
     "moment": "^2.22.2",
     "parse-link-header": "^1.0.1",
     "request": "^2.87.0",
     "request-promise-native": "^1.0.5",
     "ws": "^5.2.2"
   },
   "devDependencies": {
-    "express": "^4.16.3"
+    "express": "^4.16.3",
+    "jasmine": "^2.99.0"
```

Comment:
I updated jasmine as the previous version was throwing errors in failed async tests, which resulted in uncaught node exceptions, the update catches the errors internally and fails the test gracefully